server/service/integration_mdm_test.go

```diff
@@ -8340,6 +8376,19 @@ func (s *integrationMDMTestSuite) TestInvalidGetAuthRequest() {
 func (s *integrationMDMTestSuite) TestValidGetTOC() {
 	t := s.T()
 
+	// hacky check to make sure the assets were built (they require `-tags full`
+	// when building the tests otherwise this test will always fail due to a
+	// panic in server/bindata package)
+	defer func() {
+		if panicVal := recover(); panicVal != nil {
+			s, ok := panicVal.(string)
+			if ok && strings.Contains(s, "Assets may not be used when running Fleet as a library") {
+				t.Skip("skipping, test will fail due to assets not built (requires '-tags full')")
+			}
+		}
+	}()
+	_, _ = bindata.Asset("check if assets are build")
```

Comment:
This is to prevent a test failure when running the suite locally via other means than `make test-go`, this test requires `-tag full` to be provided to pass, and it's super annoying when running e.g. `go test ./server/service` to have it fail.

Reply:
thanks! I defaulted to running my tests with `-tag full` just because of this damn test.

It's a pity that there isn't an easy way to check build tags. I don't think we should change this, but another way to do this could be to make `bindata` expose a boolean, right? like:

```
//go:build !full
// +build !full

package bindata

const SupportsAssets = false
```

Reply:
@roperzh Yeah adding something like that to the bindata package would be less hacky but since it is all generated code I opted not to change it, not sure how it would be handled by the third-party tool that generates those files.